laghos.cpp

```diff
@@ -1128,9 +1130,11 @@ int main(int argc, char *argv[])
     {
         samplerTimer.Start();
         sampler->Finalize(t, dt, S, cutoff);
-        outfile_twp << twep[rom_window] << ", ";
-        outfile_twp << cutoff[0] << ", " << cutoff[1] << ", " << cutoff[2] << ", ";
-        outfile_twp << 2*cutoff[0] << ", " << 20*cutoff[1] << ", " << 2*cutoff[2] << "\n";
+        if (myid == 0) {
```

Comment:
I see that outfile_twp is only created if we are using windows, but this if statement doesn't have a check for using windows. Should there be a check here for using windows?

Reply:
> can the print statement be moved into laghos_rom.cpp under another function, similar to PrintSingularValues, and have it be called by ROM_Sampler::Finalize? and the cutoff variable can also be moved into ROM_Sampler::Finalize? unless it is planned to have cutoff as a user-defined option from the command line.

I certainly see that this suggestion is a good idea and also I think that cutoff variables will not be a user-defined option from the command line in the future. However, we might need to introduce the user-defined coefficients that are multiplied by cutoff values to specify the number of sample points (for now it is hard-coded to 2, 20, 2) because these numbers are problem-specific and we do not have any mathematical justification for these numbers unlike cutoff values which can be justified by the singular value ratios.  This actually raises question about the current format of twp.csv file because the number of samples depends on the corresponding cutoff values. Let's discuss on this.